source/_integrations/airgradient.markdown

```
@@ -84,6 +100,67 @@ The following entities are supported:
+          title: "High CO2 Level Alert"
+          message: >
+            The CO2 level is too high at {{ states('sensor.airgradient_carbon_dioxide') }} ppm.
+            Please consider ventilating the room.
+```
+
+{% endraw %}
+
+## Known limitations
+
+The AirGradient integration currently has the following limitations:
+- The update entity is not able to install updates automatically. You will need to reboot the device manually after installing the update.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)